src/QandA/QandA.jsx

```diff
@@ -47,22 +80,17 @@ class QandA extends React.Component {
     });
   }
 
-  componentDidMount() {
-    this.fetchQuestions();
-    // this.postQuestion();
+  componentDidUpdate(prevProps) {
```

Comment:
good idea here! Glad to see this resolved your issue with the data only being rendered once on mount and not on consecutive updates.